mlflow/pyfunc/__init__.py

```diff
@@ -828,7 +851,14 @@ def _create_model_downloading_tmp_dir(should_use_nfs):
 _MLFLOW_SERVER_OUTPUT_TAIL_LINES_TO_KEEP = 200
 
 
-def spark_udf(spark, model_uri, result_type="double", env_manager=_EnvManager.LOCAL):
+@experimental
```

Comment:
Can we remove `@experimental` from `spark_udf` and instead update the docstring of `params` to indicate that it's experimental?